components/Subscribe.tsx

```diff
@@ -1,13 +1,12 @@
-import { useState, useRef } from 'react';
-import Link from 'next/link';
-import useSWR from 'swr';
+import ErrorMessage from 'components/ErrorMessage';
```

Comment:
Could we keep the imports the same? 😄

Reply:
These got reordered by some extension. Will revert them back to original format.